unified-codes/libs/ui/src/lib/components/entity/molecules/EntityDetailList.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { Property, IEntity } from '@unified-codes/data';
+import { Typography } from '../../data';
+import { Grid } from '../../layout';
+import { Link } from '../../navigation';
+import { EntityDetailCategoryItem } from './EntityDetailCategoryItem';
+import { TypographyVariant } from '@material-ui/core';
+
+export interface IExternalLink {
+  type: string;
+  url: string;
+}
+
+export interface EntityDetailListProps {
+  headerVariant?: TypographyVariant;
+  productSubCategories?: IEntity[];
+  entityProperties?: Property[];
+  externalLinks?: IExternalLink[];
+}
+
+export type EntityDetailList = React.FunctionComponent<EntityDetailListProps>;
+
+export const EntityDetailList: EntityDetailList = ({
+  headerVariant,
+  productSubCategories,
+  externalLinks,
+  entityProperties,
+}) => {
+  const headerTypography = headerVariant ?? 'h6';
+
+  return (
+    <Grid direction="column">
+      <Typography variant={headerTypography}>Forms</Typography>
+      <ul>
+        {productSubCategories?.map((category) => (
+          <EntityDetailCategoryItem entity={category} />
+        ))}
+      </ul>
+      <Typography variant={headerTypography}>Properties</Typography>
+      <ul>
+        {entityProperties?.map((item) => (
+          <li key={item.type}>
+            {item.type}: {item.value}
+          </li>
+        ))}
+      </ul>
+      <Typography variant={headerTypography}>External Links</Typography>
```

Comment:
A refactoring suggestion if the hiding of sections is a useful idea:
```
const DetailSection = ({
  headerTypography,
  items,
  title,
}: {
  headerTypography: any;
  items: Array<JSX.Element>;
  title: string;
}) => (
  <Grid item>
    <Typography variant={headerTypography}>{title}</Typography>
    <ul>{items}</ul>
  </Grid>
);

...

const properties = entityProperties?.map((item) => (
    <li key={item.type}>
      {item.type}: {item.value}
    </li>
  ));

...

<DetailSection headerTypography={headerTypography} items={properties} title="Properties" />
```

etc. same pattern for the other two. Just a thought.
const detailSection